src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
```

Comment:
1. Please make a constant field in the class for the number 100 (`private static final int UPPER_BOUND = 100` f.e.)
2. Make a field of type `Random` in this class and call `nextInt(UPPER_BOUND);` with this new field.